cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/utils/NGSISQLUtilsTest.java

```diff
@@ -0,0 +1,379 @@
+package com.telefonica.iot.cygnus.utils;
```

Comment:
License header missing

Reply:
Fixed in [527c45f](https://github.com/telefonicaid/fiware-cygnus/pull/1934/commits/527c45fe9eb4d2a49441ed9532fa068876872000)